tx-pool/src/block_assembler/mod.rs

```diff
@@ -494,13 +494,12 @@ impl BlockAssembler {
     }
 
     pub(crate) fn build_extension(snapshot: &Snapshot) -> Result<Option<packed::Bytes>, AnyError> {
-        let consensus = snapshot.consensus();
         let tip_header = snapshot.tip_header();
-
-        let candidate_number = tip_header.number() + 1;
-        let mmr_activated_epoch = consensus.hardfork_switch().mmr_activated_epoch();
-        if tip_header.epoch().minimum_epoch_number_after_n_blocks(1) >= mmr_activated_epoch {
-            let mmr_size = leaf_index_to_mmr_size(candidate_number - 1);
+        let mmr_activate = snapshot.versionbits_active(DeploymentPos::LightClient);
+        if mmr_activate {
+            // let candidate_number = tip_header.number() + 1;
+            // mmr_size = leaf_index_to_mmr_size(candidate_number - 1);
```

Comment:
please cleanup code

Reply:
Here is intentionally kept as a comment, because the original code is actually using candidate_number, but  +1 - 1 == tip_number, the intermediate process omitted, here keep the intermediate process as a comment